src/views/templates/hook/index.tpl

```diff
@@ -0,0 +1,181 @@
+            },
+            'onboarding-welcome': {},
+            'onboarding-overview': {
+                defaultParcelGet: "{$urls['onboardingOverview']['defaultParcelGet']}",
+                defaultWarehouseGet: "{$urls['onboardingOverview']['defaultWarehouseGet']}"
+            },
+            'default-parcel': {
+                getUrl: "{$urls['defaultParcel']['getUrl']}",
+                submitUrl: "{$urls['defaultParcel']['submitUrl']}"
+            },
+            'default-warehouse': {
+                getUrl: "{$urls['defaultWarehouse']['getUrl']}",
+                getSupportedCountriesUrl: "{$urls['defaultWarehouse']['getSupportedCountriesUrl']}",
+                submitUrl: "{$urls['defaultWarehouse']['submitUrl']}",
+                searchPostalCodesUrl: "{$urls['defaultWarehouse']['searchPostalCodesUrl']}"
+            },
+            'configuration': {
+                getDataUrl: "{$urls['configuration']['getDataUrl']}"
+            },
+            'system-info': {
+                getStatusUrl: "{$urls['systemInfo']['getStatusUrl']}",
+                setStatusUrl: "{$urls['systemInfo']['setStatusUrl']}"
+            },
+            'order-status-mapping': {
+                getMappingAndStatusesUrl: "{$urls['orderStatusMapping']['getMappingAndStatusesUrl']}",
+                setUrl: "{$urls['orderStatusMapping']['setUrl']}"
+            },
+            'my-shipping-services': {
+                getServicesUrl: "{$urls['myShippingServices']['getServicesUrl']}",
+                deleteServiceUrl: "{$urls['myShippingServices']['deleteServiceUrl']}"
+            },
+            'pick-shipping-service': {
+                getServicesUrl: "{$urls['pickShippingService']['getServicesUrl']}",
+                getActiveServicesUrl: "{$urls['pickShippingService']['getActiveServicesUrl']}",
+                getTaskStatusUrl: "{$urls['pickShippingService']['getTaskStatusUrl']}",
+                startAutoConfigureUrl: "{$urls['pickShippingService']['startAutoConfigureUrl']}",
+                disableCarriersUrl: "{$urls['pickShippingService']['disableCarriersUrl']}"
+            },
+            'edit-service': {
+                getServiceUrl: "{$urls['editService']['getServiceUrl']}",
+                saveServiceUrl: "{$urls['editService']['saveServiceUrl']}",
+                getTaxClassesUrl: "{$urls['editService']['getTaxClassesUrl']}",
+                getCountriesListUrl: "{$urls['editService']['getCountriesListUrl']}",
+                hasTaxConfiguration: true,
+                hasCountryConfiguration: true,
+                canDisplayCarrierLogos: true
+            }
+        };
+
+        Packlink.state = new Packlink.StateController(
+            {
+                baseResourcesUrl: "{$baseResourcesUrl}",
+                stateUrl: "{$urls['stateUrl']}",
+                pageConfiguration: pageConfiguration,
+                templates: {
+                    'pl-login-page': {
```

Comment:
Since there is an array generated in the back end controller (currently packlink.php), maybe it would be better to have here only json_encode this array instead of constructing the whole JSON object.